### ISSUE TITLE
Shorter versions of type creators like `insets`, `point` etc.

### ISSUE THREAD

**Issue by TadeasKriz** [tolo/InterfaCSS#37]
It would be nice if there was a support for a shorthand versions of these files like there is in CSS.

For example:
`point(10, 10)` => `point(10)`
`insets(10, 10, 10, 10)` => `insets(10)`
`insets(10, 0, 10, 0)` => `insets(10, 0)`

What do you think?

**Comment by tolo** [comment]
Good suggestion - shorter and more convenient ways of expressing things is always good. I'll add this in the next feature release.

**Comment by tbrannam** [comment]
From an outside perspective   Too much shorthand requires lots of rules that other authors need to know.   And it also if mirroring another spec you need to be concerned that you mirror it exactly.   I kind of liked that the markup was more precise.

**Comment by tolo** [comment]
Good point, one should of course not go crazy with adding shorthand. But as long as a shorthand form adds value and/or is in line with what exists for the standard web CSS, I think it's addition can be justified.
